### ISSUE TITLE
Sender's display name in image viewer is not readable

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#25822]
### Steps to reproduce

1. Use dark mode
1. Receive an image from someone
2. Click on it to open it in the image viewer
3. Look at the sender's display name in the upper left corner

#### What did you expect?

Their display name should have plenty of contrast against the background

#### What happened instead?

It's unreadable:

![Screenshot 2023-07-17 at 22-30-46 Element 7 Test room](https://github.com/vector-im/element-web/assets/48614497/f29d8379-cd0f-4854-a217-cea7f9be4b8c)

### Operating system

NixOS unstable

### Browser information

Firefox 115.0

### URL for webapp

develop.element.io

### Application version

Element version: 4cf4dc9a3db6-react-ba90e0b25505-js-eb7faa6c0771 Olm version: 3.2.14

### Homeserver

Not relevant

### Will you send logs?

No
